www/src/components/navigation/ColorSchemeSelect.tsx

```diff
@@ -0,0 +1,130 @@
+import { Listbox, Transition } from "@headlessui/react";
+import { Fragment, useEffect, useState } from "react";
+
+type ThemesNames = (typeof themesOptions)[number]["name"];
+
+const findThemeByName = (name: ThemesNames) => {
+  return themesOptions.find((theme) => theme.name === name);
+};
+
+const themesOptions = [
+  {
+    name: "os default",
```

Comment:
```suggestion
    name: "System",
```

To me, `System` sounds more in line with what others do, e.g.:

<img width="499" alt="Screenshot 2023-02-21 at 05 12 04" src="https://user-images.githubusercontent.com/17348/220238720-08bd0d8b-a1cf-48a9-9ead-d2f18c72e6c5.png">